sandbox/src/main/scala/example/Sandbox.scala

```diff
@@ -157,6 +182,8 @@ enum Msg:
   case Log(msg: String)
   case WebSocketStatus(status: Status)
   case Clear
+  case OverwriteModel(model: Model)
+  case TakeSnapShot
```

Comment:
nitpicking: maybe TakeSnapshot (lower case shot) for consistency (compare HotReload.snapshot)

Reply:
Done, thanks.